src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +74,18 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should adjust the props for source', () => {
```

Comment:
Are you testing functionality of the ReactJS library? Or are you testing for a certain OCL use case?

Reply:
If I follow you correctly. You want an explanation of what happens when the test runs as per OCL. I have implemented that